src/Files.App/Views/PaneHolderPage.xaml

```diff
@@ -126,7 +126,7 @@
 				<icore:ChangePropertyAction
 					PropertyName="CurrentInstanceBorderThickness"
 					TargetObject="{Binding ElementName=PaneLeft}"
-					Value="0,0,0,1" />
+					Value="0,0,0,0" />
```

Comment:
Is this related and if so, how does this affect the Statusbar?

Reply:
It’s an unrelated fix, I can try to send the before and after…

Reply:
Just wanted to make sure, I understand this change correctly and that I didnt miss anything.

Reply:
Actually, this change is related.

The border is handled on a different control now so we no longer apply it here.